data-esnext.js

```diff
@@ -310,9 +326,10 @@ exports.tests = [
     },
     {
       name: 'SIMD.%type%.check',
+      spec: 'https://tc39.github.io/ecmascript_simd/#simd-check',
       mdn: 'https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/SIMD/check',
       exec: function(){/*
-        return typeof SIMD.Float32x4.check === 'function';
+        return typeof SIMD.Float32x4.check === 'function' && typeof SIMD.Int32x4.check === 'function' && typeof SIMD.Uint32x4.check === 'function' && typeof SIMD.Bool32x4.check === 'function';
```

Comment:
Just defining an array of strings (like `Float32x4`) at the top of the file, and reference it where needed, should do it